src/openforms/contrib/zgw/service.py

```diff
@@ -21,6 +21,7 @@ def create_zaak(
     client = config.zrc_service.build_client()
     today = date.today().isoformat()
     data = {
+        "auteur": options["author"],
```

Comment:
https://redocly.github.io/redoc/?url=https://raw.githubusercontent.com/vng-Realisatie/zaken-api/1.2.0/src/openapi.yaml#tag/zaken -> `auteur` does not exist in the request body, where did you pull this from?